controlplane/kubeadm/internal/workload_cluster_coredns.go

```diff
@@ -47,9 +47,11 @@ const (
 	coreDNSVolumeKey       = "config-volume"
 	coreDNSClusterRoleName = "system:coredns"
 
-	kubernetesImageRepository = "k8s.gcr.io"
-	oldCoreDNSImageName       = "coredns"
-	coreDNSImageName          = "coredns/coredns"
+	// KubernetesImageRepository is the default Kubernetes image repository for build artifacts.
+	KubernetesImageRepository    = "registry.k8s.io"
```

Comment:
```suggestion
	kubernetesImageRepository    = "registry.k8s.io"
```

Please only create exported constants if they are used by another package. Otherwise we can keep the API surface smaller. (like we did with other constants usually)

Reply:
same story here: I did use them used directly in the controller but since adding the `ImageRepositoryFromClusterConfig` func it does not need to get exported anymore 👍